tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -799,6 +799,20 @@ void orderAfter(ScheduleTree* root, ScheduleTree* tree, isl::union_set filter) {
   parent->insertChild(childPos, std::move(seq));
 }
 
+namespace {
+isl::union_set activeDomainPointsNoMapping(
+    const detail::ScheduleTree* root,
+    const detail::ScheduleTree* tree) {
+  auto domain = root->elemAs<detail::ScheduleTreeElemDomain>()->domain_;
+  for (auto t : tree->ancestors(root)) {
+    if (auto f = t->elemAs<detail::ScheduleTreeElemFilter>()) {
+      domain = domain.intersect(f->filter_);
+    }
+  }
+  return domain;
```

Comment:
This looks like it will return an empty set if `tree` is inside an extension.

Reply:
On Fri, Jun 15, 2018 at 01:33:06AM -0700, ftynse wrote:
> To have the domain be consistent with the domain of the function returned by `extractDom
ainToIds`

Mention that in the commit message and mention
the intersection in the comment.